zipkin-cassandra/src/schema/cassandra-schema-cql3.txt

```diff
@@ -6,6 +6,7 @@ CREATE TABLE "Zipkin"."ServiceSpanNameIndex" (
     value blob,
     PRIMARY KEY (key, column1)
 ) WITH CLUSTERING ORDER BY (column1 ASC)
+    AND COMPACT STORAGE
```

Comment:
ah ok ..  had to go read https://www.datastax.com/documentation/cql/3.0/cql/cql_reference/create_table_r.html?scroll=reference_ds_v3f_vfk_xj__using-compact-storage

might be clearer to split this into a separate commit, and describe the issue (thrift API requires this legacy storage method)